src/Backend/Core/Engine/Header.php

```diff
@@ -324,9 +324,11 @@ public function parseJS()
         if (!empty($existingJSFiles)) {
             // some files should be cached, even if we don't want cached (mostly libraries)
             $ignoreCache = array(
-                '/src/Backend/Core/Js/jquery/jquery.js',
-                '/src/Backend/Core/Js/jquery/jquery.ui.js',
-                '/src/Backend/Core/Js/ckeditor/jquery.ui.dialog.patch.js',
+                '/bower_components/jquery/dist/jquery.min.js',
+                '/bower_components/jquery-migrate/jquery-migrate.min.js',
+                '/bower_components/jquery-ui/jquery-ui.min.js',
+                '/bower_components/bootstrap/dist/js/bootstrap.min.js',
+                '/src/Backend/Core/Js/jquery/jquery.ui.dialog.patch.js',
```

Comment:
I thought you removed all jquery ui dialogs? This patch is probably not needed anymore?